cfme/tests/v2v/test_post_migrations.py

```diff
@@ -1,191 +1,97 @@
     pytest.mark.provider(
-        classes=[RHEVMProvider],
+        classes=[RHEVMProvider, OpenStackProvider],
         selector=ONE_PER_VERSION,
-        required_flags=['v2v'],
+        required_flags=["v2v"],
         scope="module"
     ),
     pytest.mark.provider(
         classes=[VMwareProvider],
         selector=ONE_PER_TYPE,
-        fixture_name='source_provider',
-        required_flags=['v2v'],
+        required_flags=["v2v"],
+        fixture_name="source_provider",
         scope="module"
     )
 ]
 
 
-def get_migrated_vm_obj(src_vm_obj, target_provider):
-    """Returns the migrated_vm obj from target_provider."""
-    collection = target_provider.appliance.provider_based_collection(target_provider)
-    migrated_vm = collection.instantiate(src_vm_obj.name, target_provider)
-    return migrated_vm
-
-
-@pytest.mark.parametrize(
-    "form_data_vm_obj_single_datastore", [["nfs", "nfs", rhel7_minimal]], indirect=True
-)
-def test_migration_policy_tag(request, appliance, v2v_providers, host_creds, conversion_tags,
-                              form_data_vm_obj_single_datastore, soft_assert):
-    """Test policy to prevent source VM from starting if migration is complete
+@pytest.mark.tier(1)
+def test_migration_post_attribute(
+        appliance, v2v_provider_setup, mapping_data_vm_obj_mini, soft_assert):
+    """
+    Test to validate v2v post-migrations usecases
 
     Polarion:
         assignee: ytale
         initialEstimate: 1/4h
+        caseimportance: critical
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
         casecomponent: V2V
     """
-    infrastructure_mapping_collection = appliance.collections.v2v_mappings
-    mapping = infrastructure_mapping_collection.create(form_data_vm_obj_single_datastore.form_data)
-
-    @request.addfinalizer
-    def _cleanup():
-        infrastructure_mapping_collection.delete(mapping)
-
-    migration_plan_collection = appliance.collections.v2v_plans
-    # vm_obj is a list, with only 1 VM object, hence [0]
-    vm_obj = form_data_vm_obj_single_datastore.vm_list[0]
+    target_provider = [
+        v2v_provider_setup.osp_provider
+        if v2v_provider_setup.rhv_provider is None
+        else v2v_provider_setup.rhv_provider][0]
+    src_vm_obj = mapping_data_vm_obj_mini.vm_list[0]
+    source_view = navigate_to(src_vm_obj, "Details")
+    summary = source_view.entities.summary("Properties").get_text_of("Container")
+    source_cpu, source_socket, source_core, source_memory = re.findall("\d+", summary)
+    migration_plan_collection = appliance.collections.v2v_migration_plans
     migration_plan = migration_plan_collection.create(
         name="plan_{}".format(fauxfactory.gen_alphanumeric()),
         description="desc_{}".format(fauxfactory.gen_alphanumeric()),
-        infra_map=mapping.name,
-        vm_list=form_data_vm_obj_single_datastore.vm_list,
-        start_migration=True,
+        infra_map=mapping_data_vm_obj_mini.map_obj.name,
```

Comment:
Can we get mapping name from mapping_data_vm_obj_mini.infra_mapping_data.name ?
If yes then we will not need the mapping object

Reply:
Should works since we are passing same `infra_mapping_data` to the `infrastructure_mapping_collection` while creating collection object, make sense! :+1: